<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpineDialog</class>
 <widget class="QDialog" name="SpineDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTreeView" name="boneTree">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>551</width>
     <height>271</height>
    </rect>
   </property>
   <attribute name="headerVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QSpinBox" name="xDimensions">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4096</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="yDimensions">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>4096</number>
   </property>
   <property name="value">
    <number>200</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblDimensions">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Image Dimensions:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpineExport">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Spine Export&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrowse">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>370</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Open JSON...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbExportWords">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Export words as events</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblSelectBone">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select the bone to attach the mouth to. If no file is opened, then a new skeleton will be created. </string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblAnimName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Note: Animations will be named after voice name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpineDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpineDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
